### PR TITLE
fix: add `senderTimestampMs` automatically in reactMessage

### PR DESCRIPTION
**Pull request by zennn08**
# Remove reaction message
I have tried with below code to remove reactMessage, but it doesn't work.
``` ts
const reactionMessage = {
    react: {
        text: "",
        key: message.key
    }
}
const sendMsg = await sock.sendMessage(id, reactionMessage)
```
then I found a way by adding `senderTimestampMs: Date.now()` to make it work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -411,8 +411,9 @@ const sendMsg = await sock.sendMessage(id, listMessage)
 
 const reactionMessage = {
     react: {
-        text: "💖",
-        key: message.key
+        text: "💖", // use an empty string to remove the reaction
+        key: message.key,
+	senderTimestampMs: Date.now() // this is needed if you want to remove the reaction
```
Comment (adiwajshing) [inline README.md:416]: may be worth it to just include this automatically in `src/utils/messages.ts` instead of requiring users to input?

Comment (zennn08) [reply README.md]: ok i will change it

**Comment by adiwajshing** [comment]
PR looks good, only need to lint

**Comment by zennn08** [comment]
done sir

**Pull request merged**